### DISCUSSION TITLE
Auto add Repo issues to Project backlog column

### DISCUSSION THREAD

**Discussion by JVimes** [community/community#41934] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

Is there a feature where all new issues from my repo to appear in my project's Backlog column? I think most small-to-medium user projects would benefit from that.

Suggest new (un-prioritized) issues don't appear below the vertical scroll boundary or else they'd be easily overlooked.

**Comment by 2percentsilk** [comment, marked as answer]
Is "Backlog" a value in your `Status` custom field or a different custom field? Do you think you'd want to be able to set any other values as they are added?

Currently, we have an auto-add workflow via an actions you could set up: https://github.com/actions/add-to-project
And if you're on an Enterprise plan, we have a built-in workflow: https://docs.github.com/en/enterprise-cloud@latest/issues/planning-and-tracking-with-projects/automating-your-project/archiving-items-automatically

**Comment by JVimes** [comment]
Just whatever column we use as backlog, so Status field I think. Sounds like the workflow is the feature I'm requesting, thank you.

I think the product appears underdeveloped to Free users. Might add some visibility into Enterprise features?
